.gitlab-ci.yml

```diff
@@ -1184,6 +1184,161 @@ test_full_integration:
     - cd ${CI_PROJECT_DIR}/mender
     - bash -c "bash <(curl -s https://codecov.io/bash) -Z -t ${MENDER_CODECOV_TOKEN} -F acceptance,${JOB_BASE_NAME} -s ${CI_PROJECT_DIR}/acceptance-tests-coverage"
 
+# Publish acceptance test coverage into codecov when either running tests for a
```

Comment:
Nit:
Should this be coveralls now?
```suggestion
# Publish acceptance test coverage into coveralls when either running tests for a
```

Reply:
Nice catch :+1: 

I won't commit now to let the full build complete, though. But I'll fix before merge.